### PR TITLE
Remove auto-index/category docs views/URLs

### PR DESCRIPTION
**Pull request by zuluecho9**
DOC-7645. Attempts to remove as many references to auto-index/category URLs as possible. This was focused on troubleshooting docs but a few other things, too.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/notifications/intro-notifications.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ metaDescription: "Notifications are where you integrate your New Relic One data
...
 
-For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
+For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
```
Comment (paperclypse) [inline src/content/docs/alerts-applied-intelligence/notifications/intro-notifications.mdx:15]:
```suggestion
For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples), and [Proactive Detection notifications](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
```

I'm curious why you removed the / on the first path, but not the second.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 2]
```diff
@@ -11,7 +11,7 @@ metaDescription: "Alerts and Applied Intelligence notification integrations are
...
 
-For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
+For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
```
Comment (paperclypse) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:14]:
```suggestion
For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples), and [Proactive Detection notifications](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
```

File: src/content/docs/apm/agents/java-agent/getting-started/introduction-new-relic-java.mdx [hunk 3]
```diff
@@ -162,4 +162,4 @@ The agent receives basic information about your host environment, such as operat
 
 ## Troubleshooting procedures [#troubleshoot]
 
-If you encounter problems with the Java agent, see the [troubleshooting documentation](/docs/apm/agents/java-agent/troubleshooting).
+If you have problems, see [No data appears](/docs/apm/agents/java-agent/troubleshooting/no-data-appears-java) and other troubleshooting docs in that section.
```
Comment (paperclypse) [inline src/content/docs/apm/agents/java-agent/getting-started/introduction-new-relic-java.mdx:165]: "have" is so much better than "encounter" :D

File: src/content/docs/apm/agents/java-agent/troubleshooting/gather-troubleshooting-information-java.mdx [hunk 4]
```diff
@@ -12,7 +12,7 @@ redirects:
   - /docs/agents/java-agent/troubleshooting/providing-troubleshooting-information-java
 ---
 
-If none of the New Relic Java agent's [troubleshooting procedures](/docs/apm/agents/java-agent/troubleshooting) resolve the problem, use the [log files](#logfiles) to troubleshoot the issue. If you need further assistance, gather the following information for getting support at [support.newrelic.com](https://support.newrelic.com).
+If the Java agent's troubleshooting procedures (like [No data appears](/docs/apm/agents/java-agent/troubleshooting/no-data-appears-java) and other docs in that category) have not resolved your problem, use the [log files](#logfiles) to troubleshoot the issue. If you need further assistance, contact [support.newrelic.com](https://support.newrelic.com).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/java-agent/troubleshooting/gather-troubleshooting-information-java.mdx:15]:
```suggestion
If the Java agent's troubleshooting procedures (like [No data appears](/docs/apm/agents/java-agent/troubleshooting/no-data-appears-java) and other docs in that category) haven't resolved your problem, use the [log files](#logfiles) to troubleshoot the issue. If you need further assistance, contact [support.newrelic.com](https://support.newrelic.com).
```

File: src/content/docs/apm/agents/net-agent/getting-started/introduction-new-relic-net.mdx [hunk 5]
```diff
@@ -82,7 +82,7 @@ When you are ready to install, use our launcher, or see the install instructions
   To stay up-to-date with new features and bug fixes, see the [.NET agent release notes](/docs/release-notes/agent-release-notes/net-release-notes).
 </Callout>
 
-After you install the agent and wait a few minutes for your app to generate traffic, data will appear in the [APM **Summary** page](/docs/apm/applications-menu/monitoring/apm-overview-page). If [no data appears](/docs/agents/net-agent/troubleshooting/no-data-appears-net), or if you encounter other problems, see New Relic's [.NET agent troubleshooting procedures](/docs/agents/net-agent/troubleshooting).
+After you install the agent and wait a few minutes for your app to generate traffic, data will appear in the [APM **Summary** page](/docs/apm/applications-menu/monitoring/apm-overview-page). If you have problems, see [No data appears](/docs/agents/net-agent/troubleshooting/no-data-appears-net) and the other troubleshooting docs in that section.
```
Comment (paperclypse) [inline src/content/docs/apm/agents/net-agent/getting-started/introduction-new-relic-net.mdx:85]: I think you could use "...and other troubleshooting docs..." instead (drop the "the"), but it's only a slight preference.

File: src/content/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs.mdx [hunk 6]
```diff
@@ -116,13 +116,13 @@ You can also bring your logs and application's data together to make troubleshoo
...
+If you encounter issues with your Node.js agent, see our troubleshooting docs. Some of the most important troubleshooting docs in that section include:
 
 * [Large memory usage](/docs/agents/nodejs-agent/troubleshooting/troubleshooting-large-memory-usage-nodejs): If you've installed the Node.js agent and your memory usage has increased, check out these possible solutions.
 * [Troubleshooting your Node.js installation](/docs/agents/nodejs-agent/troubleshooting/troubleshooting-your-nodejs-installation): Try these steps if you don't see any data, cannot log files, or encounter other installation problems with the Node.js agent.
 * [Troubleshooting browser instrumentation](/docs/agents/nodejs-agent/troubleshooting/troubleshooting-page-load-timing-nodejs): If you encounter problems with browser data, see these additional tips for Node.js.
 
-You can also [view all troubleshooting docs](/docs/agents/net-agent/troubleshooting). If you need additional assistance, get support at [support.newrelic.com](https://support.newrelic.com).
+If you need additional assistance, get support at [support.newrelic.com](https://support.newrelic.com).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs.mdx:125]:
```suggestion
If you need more help, get support at [support.newrelic.com](https://support.newrelic.com).
```

File: src/content/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs.mdx [hunk 7]
```diff
@@ -116,13 +116,13 @@ You can also bring your logs and application's data together to make troubleshoo
 
 ## Troubleshoot your installation [#troubleshooting]
 
-If you encounter issues with your Node.js agent, see our troubleshooting information:
+If you encounter issues with your Node.js agent, see our troubleshooting docs. Some of the most important troubleshooting docs in that section include:
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs.mdx:119]: I like this colon phrase a lot

File: src/content/docs/apm/agents/php-agent/getting-started/introduction-new-relic-php.mdx [hunk 8]
```diff
@@ -146,7 +146,7 @@ After installing the agent, go further and extend the agent's instrumentation:
 
 ## Troubleshoot your installation [#troubleshoot]
 
-If you encounter issues with the PHP agent, see our [full list of troubleshooting documentation](/docs/agents/php-agent/troubleshooting). Common installation issues include:
+If you're having problems, see the PHP agent troubleshooting docs. Common issues include:
```
Comment (paperclypse) [inline src/content/docs/apm/agents/php-agent/getting-started/introduction-new-relic-php.mdx:149]: I prefer what you've got in the Node.js doc: "Some of the most important troubleshooting docs in that section include:"

File: src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-infrastructure/no-data-appears-infrastructure.mdx [hunk 9]
```diff
@@ -177,3 +177,7 @@ If you are missing data from an integration, see troubleshooting procedures for:
 * [APM data missing from infrastructure monitoring](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure)
 * [Amazon/AWS integrations](/docs/infrastructure/amazon-integrations/troubleshooting/no-data-appears-amazonaws-integrations)
 * [On-host integrations](/docs/infrastructure/on-host-integrations/troubleshooting/not-seeing-host-integration-data)
+
+## Other factors affecting access [#more-info]
+
+For more on factors that can affect your ability to access New Relic features, see [Factors affecting access](/docs/accounts/accounts-billing/general-account-settings/factors-affecting-access-features-data).
```
Comment (paperclypse) [inline src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-infrastructure/no-data-appears-infrastructure.mdx:183]: Seems like "Factors" in the link text should be lowercase

File: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations.mdx [hunk 10]
```diff
@@ -20,7 +20,7 @@ The Azure integrations are not the same as [APM's .NET support for Azure](/docs/
 
 ## Requirements [#permissions]
 
-Check the [Azure integrations documentation](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list) for requirements on individual integrations.
+Check the docs for our [Azure integrations](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list) for requirements for specific integrations.
```
Comment (paperclypse) [inline src/content/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations.mdx:23]:
```suggestion
See [Azure integrations](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list) for requirements on specific integrations.
```

File: src/content/docs/logs/troubleshooting/no-log-data-appears-ui.mdx [hunk 11]
```diff
@@ -103,3 +103,8 @@ If no data appears after you send some log payloads and wait about five minutes,
     </tr>
   </tbody>
 </table>
+
+## Other factors affecting access [#more-info]
+
+For more on factors that can affect your ability to access New Relic features, see [Factors affecting access](/docs/accounts/accounts-billing/general-account-settings/factors-affecting-access-features-data).
```
Comment (paperclypse) [inline src/content/docs/logs/troubleshooting/no-log-data-appears-ui.mdx:109]: Seems like "Factors" in the link text should be lowercase

File: src/content/docs/new-relic-partnerships/partner-integration-guide/getting-started/support-resources-new-relic-partners.mdx [hunk 12]
```diff
@@ -11,10 +11,10 @@ redirects:
   - /docs/accounts-partnerships/partner-integration-guide/getting-started/support-resources-new-relic-partners
 ---
 
-This document is for New Relic partners. After you complete your partnership integration, you will be introduced to your technical contacts at New Relic. Use these channels for non-urgent escalations.
+This document is for [New Relic partners](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partnership-accounts-users-subscriptions). After you complete your partnership integration, you'll be introduced to your technical contacts at New Relic. Use these channels for non-urgent escalations.
```
Comment (paperclypse) [inline src/content/docs/new-relic-partnerships/partner-integration-guide/getting-started/support-resources-new-relic-partners.mdx:14]:
```suggestion
This document is for our [New Relic partners](/docs/new-relic-partnerships/partner-integration-guide/partner-account-maintenance/partnership-accounts-users-subscriptions). After you complete your partnership integration, you'll be introduced to your technical contacts at New Relic. Use these channels for non-urgent escalations.
```

**Review by paperclypse — APPROVED** [review]
Looks good! Left some minor comments/suggestions.

File: src/content/docs/alerts-applied-intelligence/notifications/intro-notifications.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/alerts-applied-intelligence/notifications/intro-notifications.mdx:15]: just removing them as I see them

**Pull request merged**